spec/rails_helper.rb

```diff
@@ -61,4 +61,9 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  VCR.configure do |config|
+    config.cassette_library_dir = "fixtures/vcr_cassettes"
+    config.hook_into :webmock
+    config.allow_http_connections_when_no_cassette = true
+  end
```

Comment:
It looks like VCR is not currently being used for any of your tests. You're probably aware of this, but for your tests to run with this VCR configuration they need a `:vcr` flag in the example definition.